ui/appui-react/src/appui-react/layout/tool-settings/Handle.scss

```diff
@@ -4,20 +4,14 @@
  *--------------------------------------------------------------------------------------------*/
 
 .nz-toolSettings-handle {
```

Comment:
Should we replace this Handle component with iTwinUI borderless button?

Reply:
I think if we used a button it would be expected that a click OR drag & drop would undock the tool settings. I'm thinking maybe we revisit this decision in a future PR where we try to address #727 .  What do you think?

Reply:
Sounds good, let's do that.